### PR TITLE
Feat/45 setup vue testing library

### PR DESCRIPTION
**Pull request by nickbouldien**
some progress here, but will need a decent amount of work to get everything tested.  But I think this now allows people to write some tests, though I think one big thing is that we need to mock the Airtable stuff which means it might need to be extracted out and then used in the components.  That might take a bit of time, so I'm trying to get some basic setup in with this PR

also, did a tad bit of refactoring the `ContactForm` component when doing this.  Tried to use a bit more clear/accessible naming

EDIT: forgot to mention - this shouldn't affect functionality at all and you can run tests (the very few that exist) with `yarn test`

### CODE REVIEW

File: src/__tests__/ContactForm.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  // buttons
+  const submitButton = getByText("Submit");
+  expect(submitButton).toBeTruthy();
+
+  const resetButton = getByText("Reset");
+  expect(resetButton).toBeTruthy();
+});
+
+test('Check the reset button clears the input', async () => {
```
Comment (pjhoberman) [inline src/__tests__/ContactForm.test.js:42]: Is this testing reset or submit?

**Review by pjhoberman — DISMISSED** [review]
This is awesome. Thank you! Left one question but not a blocker.

**Comment by pjhoberman** [comment]
Closes #45

File: src/__tests__/ContactForm.test.js (hunk 1 above)
Comment (nickbouldien) [reply src/__tests__/ContactForm.test.js]: good call. forgot to change that

**Review by pjhoberman — APPROVED** [review]

**Pull request merged**